### ISSUE TITLE
pelican-quickstart is not properly escaping paths

### ISSUE THREAD

**Issue by Chaostheorie** [getpelican/pelican#2803]
- [X] I have read the [Filing Issues](https://docs.getpelican.com/en/latest/contribute.html#filing-issues) and subsequent “How to Get Help” sections of the documentation.
- [X] I have searched the [issues](https://github.com/getpelican/pelican/issues?q=is%3Aissue) (including closed ones) and believe that this is not a duplicate.

- **OS version and name**: Ubuntu 20.04.1 LTS x86_64
- **Python version**: Python 3.8.2
- **Pelican version**: 4.5.0
- **Links to plugins**: jinja2content, uglify

## Issue
The `Makefile` generated by pelican-quickstart doesn't escape file paths by default. This may lead to bash or sh interpreting them as multiple arguments, e.g. 

Makefile (excerpt): ` $(PELICAN) -l $(INPUTDIR) -o $(OUTPUTDIR) -s $(CONFFILE) $(PELICANOPTS)`

Will be executed as: `pelican /home/cobalt/P/…/content -o /home/cobalt/…/public -s /home/cobalt/…/pelicanconf.py ` Which leads to `pelican: error: unrecognized arguments: …/pelicanconf.py`

This can be escaped by adding `"$(VARIABLE)"` around the Variables in the `Makefile`:
```Makefile
html:
        $(PELICAN) $(INPUTDIR) -o $(OUTPUTDIR) -s $(CONFFILE) $(PELICANOPTS)
->
html:
        $(PELICAN) "$(INPUTDIR)" -o "$(OUTPUTDIR)" -s "$(CONFFILE)" $(PELICANOPTS)
```

**Comment by avaris** [comment]
This was fixed in #2542. Did you create a Makefile with `pelican 4.5.0` or are you looking at a Makefile created by a previous version of `pelican`?

**Comment by Chaostheorie** [comment]
I've used the build from the Ubuntu (uni- or multiverse) repositories. Could have been not properly replaced. But this is a problem on my system. Falls to hear it's fixed
